package.json

```diff
@@ -51,26 +51,15 @@
     }
   },
   "dependencies": {
-    "agentkeepalive": "^2.2.0",
+    "agentkeepalive": "^4.1.3",
     "axios": "^0.19.0",
-    "es6-promise": "^4.2.6",
-    "json-stringify-safe": "^5.0.1",
-    "lodash.bind": "^4.2.1",
-    "lodash.clonedeep": "^4.5.0",
-    "lodash.foreach": "^4.5.0",
-    "lodash.isequal": "^4.5.0",
-    "object-sizeof": "^1.0.10",
-    "valid-url": "^1.0.9"
+    "json-stringify-safe": "^5.0.1"
   },
   "devDependencies": {
-    "assert": "^1.4.1",
-    "delay": "^4.1.0",
     "eslint": "^7.4.0",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-sensible": "^2.2.0",
-    "mocha": "^5.2.0",
     "nock": "^13.0.2",
-    "nyc": "^14.1.1",
-    "tap": "^14.10.7"
+    "tap": "^14.10.8"
   }
```

Comment:
Just a note for this file—whether or not we're bumping the version to `4.0.0` in this commit, or a separate PR.  Auto versioning is not yet set up for this repo.